### PR TITLE
Allstar

### PR DESCRIPTION
**Pull request by EjiroLaurelD**
Hello, my name is Laurel , an Outreachy applicant. 
I have investigated what Allstar is about, it's capabilities and how it aligns with the needs of this project as indicated in the checklist in issue #12. 
I found out that Allstar is a Security Policy for open source projects which can help to ensure that all repositories have a security policy, scan all repos for common vulnerabilities, enforce best practices for code reviews, and require security testing for all releases. 
I have written a documentation which highlights; 
How much of the checklist Allstar can cover
What items needs to be manually configured and 
Proposed steps to enable Allstar across organizations.
Please let me know if i need to fix anything to make it a better contribution. Thank you

### CODE REVIEW

**Review by jpkrohling — COMMENTED** [review]
Who's the audience for this? If this is for other maintainers, the last section doesn't seem relevant. If it's for other sig-security members, it would be good to use this to document how we use allstar.

File: allstar.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+I have investigated what Allstar is about, it's capabilities and how it aligns with the needs of this project as inidicated in the checklist in issue #12.
```
[the remark this answers is not in the document]
Comment (EjiroLaurelD) [reply allstar.md:1]: Okay, I will fix that. Thank you for pointing it out

File: allstar.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Allstar is a security policy engine that helps organizations automate and enforce security best practices. Allstar can be used to scan code, dependencies, andinfrastructure for vulnerabilities. Allstar can also be used to enforce best practices for code reviews, security testing, and vulnerability management.
```
[the remark this answers is not in the document]
Comment (EjiroLaurelD) [reply allstar.md:2]: I'll fix that now. thank you

**Comment by jpkrohling** [comment]
As discussed on #21, I'm closing this one.

**Pull request closed without merging**